### PR TITLE
Implement tokenize method based off of class example

### PR DESCRIPTION
**Pull request by cbolton97**
- Used the guide in the [lecture slides](https://docs.google.com/presentation/d/1YiDDOYj-fAbMtAyYmoRVwZcmNrwTfi9e0i4jfnAocT4/edit#slide=id.g40cccd4300_0_567) to implement the tokenize method
- Commented out some code on `main.ts` in order to allow the program to run
- Try it out by running `yarn install && yarn run build && yarn run calendar` (you'll get errors after the build, that's okay)

### CODE REVIEW

File: src/Tokenizer.ts [hunk 1]
```diff
@@ -1,45 +1,93 @@
...
+    this.tokenize();
   }
 
-  checkToken(regex): boolean {
-    s: String = checkNext();
-    return (s.matches(regexp));
-  }
+  private tokenize() {
+    // Remove all newlines
+    let tokenizedProgram = this.program.replace(/\r?\n|\r/g, "");
```
Comment (shannonjanehogan) [inline src/Tokenizer.ts]: Not sure I totally understand this part- if we remove all newlines, will we still be able to report errors with the line number of the code that caused the error?

File: src/Tokenizer.ts [hunk 2]
```diff
@@ -1,45 +1,93 @@
...
+    "Events:",
+    "every",
+    "and",
+    "on",
+    "all day",
+    "from",
+    "at",
+    "with",
+    "Done",
+    "End"
```
Comment (shannonjanehogan) [inline src/Tokenizer.ts]: lols this was a good call

**Comment by shannonjanehogan** [comment]
Overall looks great! Just one part I was confused about

File: src/Tokenizer.ts (hunk 1 above)
Comment (cbolton97) [reply src/Tokenizer.ts]: Oh good point.

**Pull request merged**